src/rs/ffi/win_bindings.rs

```diff
@@ -5807,50 +5852,6 @@ const _: () = {
 };
 #[repr(C)]
 #[derive(Debug, Copy, Clone)]
-pub struct QUIC_CONNECTION_EVENT__bindgen_ty_1__bindgen_ty_19 {
```

Comment:
In the Rust wrappers u need to change the conversion function to access the newly named struct instead of the old anonymous struct.